docs/managing-compute-resources.asciidoc

```diff
@@ -114,4 +114,4 @@ spec:
             limits: {}
 ----
 
-The default `requests` will not be set by the operator and the Pod will be created.
\ No newline at end of file
+The default `requests` will not be set by the operator and the Pod will be created.
```

Comment:
This is a side effect of running `sed` on OSX: it will add a new line where missing. Not expected but actually useful in this case 🤷‍♂